### PR TITLE
Pref resource legacy tests

### PR DESCRIPTION
**Pull request by nmonterroso**
@Wikia/services-team 
adding integration test for legacy resource to make sure it can be called alongside the new resource

### CODE REVIEW

File: service/user-preference/src/test/java/com/wikia/userpreference/resources/UserPreferenceResourceIntegrationTest.java [hunk 1]
```diff
@@ -31,11 +32,13 @@
 import org.junit.experimental.categories.Category;
 
 import java.io.InputStream;
```
Comment (WikiaTech) [inline service/user-preference/src/test/java/com/wikia/userpreference/resources/UserPreferenceResourceIntegrationTest.java]: MINOR Remove this unused import 'java.io.InputStream'. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)

File: service/user-preference/src/test/java/com/wikia/userpreference/resources/UserPreferenceResourceIntegrationTest.java [hunk 2]
```diff
@@ -14,6 +14,7 @@
 import com.wikia.test.annotation.IntegrationTest;
 import com.wikia.userpreference.UserPreferenceApplication;
 import com.wikia.userpreference.configuration.UserPreferenceConfiguration;
+import com.wikia.userpreference.domain.Preference;
 import com.wikia.userpreference.domain.UserPreferences;
 
 import com.fasterxml.jackson.core.type.TypeReference;
```
Comment (WikiaTech) [inline service/user-preference/src/test/java/com/wikia/userpreference/resources/UserPreferenceResourceIntegrationTest.java]: MINOR Remove this unused import 'com.fasterxml.jackson.core.type.TypeReference'. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues:
* MAJOR 1 major
* MINOR 3 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Refactor this method to throw at most one checked exception instead of: java.sql.SQLException, java.net.URISyntaxException](https://github.com/Wikia/pandora/blob/03a5e299a3b2ae2f9084e9b7627f81e310a04a8a/service/user-preference/src/main/java/com/wikia/userpreference/resources/UserPreferenceResource.java#L74) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1160)
* MINOR [Remove this unused import 'com.fasterxml.jackson.databind.ObjectMapper'.](https://github.com/Wikia/pandora/blob/03a5e299a3b2ae2f9084e9b7627f81e310a04a8a/service/user-preference/src/test/java/com/wikia/userpreference/resources/UserPreferenceResourceIntegrationTest.java#L21) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by ArturKlajnerok** [comment]
Lgtm :+1:

**Pull request merged**